### PR TITLE
Add recipe for neurofire, a deep learning toolkit for connectomics

### PR DESCRIPTION
**Pull request by constantinpape**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by constantinpape** [comment]
@conda-forge/help-python
I think this is good to be reviewed.

I don't quite understand why all CIs are triggered though, because this is a noarch package.

File: recipes/neurofire/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% set version = "v0.1.0" %}
```
Comment (npavlovikj) [inline recipes/neurofire/meta.yaml]: This should be `{% set version = "0.1.0" %}`.

File: recipes/neurofire/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+{% set checksum = "9f23eb77529fc34520c1a79022833f024cd4acb52ba406d08dc4d191372a7343" %}
+
+package:
+    name: neurofire
+    version: {{ version }}
+
+source:
+    url: https://github.com/inferno-pytorch/neurofire/archive/{{ version }}.tar.gz
```
Comment (npavlovikj) [inline recipes/neurofire/meta.yaml]: This should be `url: https://github.com/inferno-pytorch/neurofire/archive/v{{ version }}.tar.gz`.

File: recipes/neurofire/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    sha256: {{ checksum }}
+
+build:
+    number: 0
+    script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (npavlovikj) [inline recipes/neurofire/meta.yaml]: This is enough to be: `script: "{{ PYTHON }} -m pip install . -vv"`.

File: recipes/neurofire/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    noarch: python
+
+requirements:
+    host:
+        - python >=3.6
+        - pip
+
```
Comment (npavlovikj) [inline recipes/neurofire/meta.yaml]: Please remove empty line.

File: recipes/neurofire/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
+{% set version = "v0.1.0" %}
+{% set checksum = "9f23eb77529fc34520c1a79022833f024cd4acb52ba406d08dc4d191372a7343" %}
```
Comment (npavlovikj) [inline recipes/neurofire/meta.yaml]: Just for consistency, I would use `sha256` instead of `checksum`.

File: recipes/neurofire/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+package:
+    name: neurofire
```
Comment (npavlovikj) [inline recipes/neurofire/meta.yaml]:
```
name: {{ name|lower }}
```

File: recipes/neurofire/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+    version: {{ version }}
+
+source:
+    url: https://github.com/inferno-pytorch/neurofire/archive/{{ version }}.tar.gz
+    sha256: {{ checksum }}
```
Comment (npavlovikj) [inline recipes/neurofire/meta.yaml]:
```
sha256: {{ sha256 }}
```

**Comment by npavlovikj** [comment]
@constantinpape , I left you few comments. You are also deleting `recipes/example/meta.yaml `. Please revert this and DO NOT delete files that are not part of your recipe directory.

**Comment by npavlovikj** [comment]
> @conda-forge/help-python
> I think this is good to be reviewed.
> 
> I don't quite understand why all CIs are triggered though, because this is a noarch package.

`noarch` recipes are only built on Linux, so failures on other platforms are ok.

**Comment by constantinpape** [comment]
Thanks for the review @npavlovikj! I have pushed a new commit that should address all your comments.

File: recipes/neurofire/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,38 @@
+{% set version = "0.1.0" %}
+{% set sha256 = "9f23eb77529fc34520c1a79022833f024cd4acb52ba406d08dc4d191372a7343" %}
+
```
Comment (npavlovikj) [inline recipes/neurofire/meta.yaml]: Please add `{% set name = "neurofire" %}` here (sorry I wasn't more specific about this earlier).

Comment (constantinpape) [reply recipes/neurofire/meta.yaml]: Oh, and I thought there was some magic infering the name ;).
Anyway, I added the name variable now.

Comment (npavlovikj) [reply recipes/neurofire/meta.yaml]: Ah, no, it is a variable we define - I am just so used of using them all the time :)

**Comment by npavlovikj** [comment]
Thank you for contributing and the prompt fixes @constantinpape 🎉

**Comment by constantinpape** [comment]
Thanks a lot for the fast and thorough review @npavlovikj  :).

**Pull request merged**